src/TempleLineOfCredit.sol

```diff
@@ -0,0 +1,318 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.13;
+
+
+import {Ownable} from "openzeppelin-contracts/access/Ownable.sol";
+import {IERC20} from "openzeppelin-contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "openzeppelin-contracts/token/ERC20/utils/SafeERC20.sol";
+
+import {Operators} from "./common/access/Operators.sol";
+
+contract TempleLineOfCredit is Ownable, Operators {
+
+    using SafeERC20 for IERC20;
+
+    struct Position {
+        uint256 collateralAmount;
+        uint256 debtAmount;
+        uint256 createdAt;
+    }
+
+    // Collateral Parameters
+
+    /// @notice Supported collateral token address
+    IERC20 public immutable collateralToken;
+
+    /// @notice Collateral token price
+    uint256 public collateralPrice;
+
+    /// @notice Requited collateral backing to not be in bad debt in percentage
```

Comment:
typo - should be "Required"

So to get 95%, you enter `95`? Or some other precision? Worth adding to the comment imo.

If that's right, that means there's no possibility for 0.5% if that was required? Worth making as 10_000 precision just in case?

Reply:
yeah doesn't hurt to increase percision